eng/testing/performance/microbenchmarks.proj

```diff
@@ -148,10 +148,10 @@
       <PayloadDirectory>$(WorkItemDirectory)</PayloadDirectory>
       <PreCommands Condition="'$(Compare)' == 'true'">$(WorkItemCommand) --bdn-artifacts $(BaselineArtifactsDirectory) --bdn-arguments="--anyCategories $(BDNCategories) $(ExtraBenchmarkDotNetArguments) $(BaselineCoreRunArgument)"</PreCommands>
       <Command Condition="'$(AGENT_OS)' != 'Windows_NT'">
-        if [ "x$PERF_PREREQS_INSTALLED" = "x1" ]; then
-          $(WorkItemCommand) --bdn-artifacts $(ArtifactsDirectory) --bdn-arguments="--anyCategories $(BDNCategories) $(ExtraBenchmarkDotNetArguments) $(CoreRunArgument)";
-        else
+        if [ "x$PERF_PREREQS_INSTALL_FAILED" = "x1" ]; then
           echo "\n\n** Error: Failed to install prerequisites **\n\n"; export _commandExitCode=1;
```

Comment:
This line should also exit 1, not use the export command

Reply:
good catch!